templates/about.html

```diff
@@ -0,0 +1,61 @@
+{% extends 'base_layout.html' %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1WcNN2RlQQrbLSP5ZgcIsAeQAHj5h_yDW6Yd5oBMmU3M{% endblock meta_copydoc %}
+
+{% block meta_description %}Charmhub description{% endblock %}
+
+{% block content %}
+<section class="p-strip--charmhub is-shallow is-dark">
+  <div class="u-fixed-width">
+    <h1 class="p-heading--2">This is the about page</h1>
```

Comment:
Surely "About Open Operators Collection" is better?

Reply:
@anthonydillon this about page is more a placeholder for now. Daniel is looking at it and will properly implement it after he's done.